docs/recording.md

```diff
@@ -0,0 +1,37 @@
+# Call recording API
+
+* API v1: 🏁 since Nextcloud 26
```

Comment:
Finish flag means it's dead/done/stopped/not-working anymore.
Following `call.md` you want to use the following:

```suggestion
* API v1: Base endpoint `/ocs/v2.php/apps/spreed/api/v1`: since Nextcloud 26
```